golang/outbound.go

```diff
@@ -170,13 +175,18 @@ func (call *OutboundCall) Response() *OutboundCallResponse {
 }
 
 // createStatsTags creates the common stats tags, if they are not already created.
-func (call *OutboundCall) createStatsTags(connectionTags map[string]string) {
+func (call *OutboundCall) createStatsTags(connectionTags map[string]string, callOptions *CallOptions, operation string) {
 	call.commonStatsTags = map[string]string{
 		"target-service": call.callReq.Service,
 	}
 	for k, v := range connectionTags {
 		call.commonStatsTags[k] = v
 	}
+
+	if callOptions.Format != HTTP {
+		call.commonStatsTags["target-endpoint"] = string(operation)
+	}
+
```

Comment:
nit: delete blank, go style normally avoids new lines after a `{` and before a `}`